zentral/contrib/inventory/forms.py

```diff
@@ -59,11 +59,28 @@ class MetaBusinessUnitSearchForm(forms.Form):
 
 
 class MetaBusinessUnitForm(forms.ModelForm):
+
+    api_enrollment = forms.BooleanField(label="API Enrollment", required=False,)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.pk and self.instance.api_enrollment_enabled():
+            self.fields['api_enrollment'].initial = True
+            self.fields['api_enrollment'].disabled = True
+
     class Meta:
         model = MetaBusinessUnit
-        fields = ("name",)
+        fields = ("name", "api_enrollment")
```

Comment:
api_enrollment is not a field on the model.

Reply:
No is not, but is needed in the view.